### ISSUE TITLE
Accessibility - talkback Android does not work

### ISSUE THREAD

**Issue by nozem79** [signalapp/Signal-Android#14551]
### Guidelines

- [x] I have searched searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] This isn't a feature request or a discussion topic

### Bug description

Talkback (accessibility function for Android), does not work anymore. A new that it works once..

I have to telephones on Samsung and one Pixel, and on both it does not work anymore.

So blind and otherwise impaired people cannot use Signal.

### Device

Pixel 7 pro

### Android version

Android 16

**Comment by alex-signal** [comment]
Hi! Did this just happen this last release? Are you able to provide a debug log?

**Comment by nozem79** [comment]
no, it doesn't work for a long time. you can try it yourself by enabling the 'select to speach' function in accessibility.

Verzonden vanaf Outlook voor Android<https://aka.ms/AAb9ysg>
________________________________

**Comment by aryanchoudharypro** [comment]
Hi @alex-signal Would signal except a pr if i add talkback actions to the areas that need them, as well as unify the message reading logic for talkback so its consistent with other messageing apps that screen reader users are used to? Eg currently, i have to swipe 2 or 3 times to go to the next message. Let me know if i should've created a new issue about this, I asked here as this issue was also related to screenreader users on android.
